github_connector/models/github_repository_branch.py

```diff
@@ -193,11 +193,11 @@ def _get_analyzable_files(self, existing_folder):
         return res
 
     @api.model
-    def analyze_code_one(self, branch):
+    def analyze_code_one(self):
         """Overload Me in custom Module that manage Source Code analysis.
         """
         self.ensure_one()
-        path = branch.local_path
+        path = self.local_path
```

Comment:
For this, you should change `@api.model` to `@api.multi`

Reply:
Totally, but `self` was used previously, so it's very inconsistent. Removing `@api.model`...

Reply:
Changed